test/websocket/utf8_checker.cpp

```diff
@@ -24,7 +24,7 @@ class utf8_checker_test : public beast::unit_test::suite
     testOneByteSequence()
     {
         utf8_checker utf8;
-        std::array<std::uint8_t, 256> const buf =
+        std::array<std::uint8_t, 256> buf =
```

Comment:
General comment for the test:

I understand that we're kind of covering this case via the loops in the 2-, 3- and 4-byte sequence checkers, but could we add an explicit test for some fixed overlong sequences?